### DISCUSSION TITLE
Legend symbols

### DISCUSSION THREAD

**Discussion by alwgnr** [GSI-CS-CO/chart-fx#426] [Q&A] [answered]
Hello together,
is it possible to change the legend symbols to circles?
I am using the ErrorDataSet.
<img width="369" alt="grafik" src="https://user-images.githubusercontent.com/2658682/120581085-7ffeaf00-c42a-11eb-9376-20dfeaf97db1.png">

The second question is related to this: How I can display the legend without the grid?
<img width="221" alt="grafik" src="https://user-images.githubusercontent.com/2658682/120582188-5a72a500-c42c-11eb-9188-7360b1d50e6e.png">

**Comment by wirew0rm** [comment, marked as answer]
Hey,

just subclass the ErrorDataSetRenderer overriding the [method which generates the legend symbol](https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/ErrorDataSetRenderer.java#L77). 

The grid in the legend is the error surface and it will not be drawn if you disable rendering it by setting the renderer's ErrorStyle to NONE (assuming your data does not contain error data that you want to have drawn). But if you override `drawLegendSymbol` you already have full control over what is drawn.

Hope this helps,

Alex
